addon/mixins/list-input.js

```diff
@@ -0,0 +1,71 @@
+// import _ from 'lodash'
+import Ember from 'ember'
+// import computed, {readOnly} from 'ember-computed-decorators'
+
+import * as utils from '../components/utils'
+
+export default Ember.Mixin.create({
+  init () {
+    this._super(...arguments)
```

Comment:
I was just thinking about that.  I'm pretty sure the mixin's `init` clobbers the mixee's `init`, and will call `_super` up the inheritance chain as normal.  This is actually an argument in favor of inheritance here.